### ISSUE TITLE
Optimization unable to find requested metric report

### ISSUE THREAD

**Issue by JoejynWan** [allegroai/clearml#285]
Hi there, 

I am trying to conduct hyperparameter optimisation with my model built from pytorch. However, whenever I run my optimise script, I get the following error: 

`clearml.automation.optimization - WARNING - Could not find requested metric ('Training', 'accuracy') report on base task ...`

I am assuming this is from my inputs of `objective_metric_title='Training'` and `objective_metric_series='accuracy'`, where in my training script, I have reported the scalars with the respective titles and series: 

```
Logger.current_logger().report_scalar(
    "Training", "loss", iteration= j, value= loss.item())
Logger.current_logger().report_scalar(
    "Training", "accuracy", iteration= j, value= (total_correct / total_count))
```

May I know if I am missing something? Thank you.

**Comment by bmartinn** [comment]
Hi @JoejynWan 
The warning is basically telling you that the Task that you are trying to optimize never reported the metrics your are looking to optimize. Basically this is trying to help with typos in metric names :)

Based on you code snippet, you are reporting the exact metric you are trying to optimize, so I can't see any issue there.

I assume that the base task we are optimizing was never executed, meaning there are no reported metrics and this is why you are getting the warning, is that correct ?

BTW: Maybe we should not display the warning if the base-task is in "draft" state?
WDYT?

**Comment by JoejynWan** [comment]
Hi @bmartinn 

I did execute the base task, but I did not execute it fully. Either ways, it worked once it was executed fully, so you are right there. 

I actually think that it is a good idea to display the warning even when the base-task is in "draft". This is because it took me a while to even figure out what I should be inputting for `objective_metric_title` and `objective_metric_series`, so the warning helped me to realise that something was wrong. 

Thank you for your help!
